common-lib/src/cmr/common/cache.clj

```diff
@@ -32,19 +34,30 @@
    [cache]
    "Returns the size of the cache in bytes."))
 
+(defn simple-cache?
+  "Function that takes a cache and checks to see if the
+   cache is cmr.common.cache or cmr.common.hash-cache."
+  [cache]
+  (if (string/includes? (str (type cache)) "hash_cache")
```

Comment:
`string/includes?` already returns true or false, if is not needed.  Consider a try/catch as this function can throw an  exception(if it doesn't make sense just disregard).

Reply:
I rewrote this using your suggestion of instance?
I don't need a try catch anymore. 
I also got rid of the if.